lib/application.js

```diff
@@ -537,14 +537,19 @@ class Application extends api.events.EventEmitter {
         callback();
         return;
       }
-      if (placeName === 'tasks') {
-        this.scheduler.setTask(sectionName, exports);
-      } else if (placeName === 'setup') {
+      if (placeName === 'setup') {
```

Comment:
Why have you swapped branches?

Reply:
Sorry, accidential change. Will revert this ASAP